app/models/piece.rb

```diff
@@ -0,0 +1,7 @@
+class Piece < ActiveRecord::Base
```

Comment:
Or are you talking about scoped associations on the game model? I feel like `@game.pieces.where(type: "Rook", color: "white")` is pretty straightforward.